sample_data_config/europepmc/europepmc.config.yaml

```diff
@@ -0,0 +1,23 @@
+europePmc:
+  - dataPipelineId: europepmc_datapeline_1
+    source:
+      # apiUrl: 'https://www.ebi.ac.uk/europepmc/webservices/rest/search'
+      apiUrl: 'https://www.ebi.ac.uk/europepmc/webservices/test/rest/search'
+      search:
+        query: 'DOI:10.1101/2021.07.05.451181 OR DOI:10.1101/2021.08.10.455713'
+        hasTextMinedTerms: 'Y'
+      fieldsToReturn:
+        - title
+        - doi
+      maxDays: 1000  # limit the number of days to process from start date
```

Comment:
Perhaps one alternative to `maxDays` would be to `update state ever n days`. The difference would be that the pipeline would always process until *yesterday*, but update the state every so often.

Reply:
Although it might be good to get it out soon.